content/concepts/multiplex/overview.md

```diff
@@ -9,24 +9,33 @@ aliases:
 
 ## Overview
 
-Stream Multiplexing (_stream muxing_) is a way of sending multiple streams of data over one
-communication link. It combines multiple signals into one unified signal so it can be transported
-'over the wires', then it is demulitiplexed (_demuxed_) so it can be output and used by separate
-applications. This is done to share a single TCP connection using unique port numbers to distinguish
-streams, between the multiple proceeses (such as kademlia and gossipsub) used by applications (such as IPFS)
-to make connection and transmission more efficient.
-
-With muxing, libp2p applications may have many separate streams of communication between peers, as well as
-have multiple concurrent streams open at the same time with a peer. Stream multiplexing allows us to initialize
-and use the same [transport](../../transports/overview) connection across the lifetime of our interaction with a
-peer. With muxing, we also only need to deal with [NAT traversal](../../nat/overview) once to be able to open as
-many streams as we need, since they will all share the same underlying transport connection. Applications can enable
-support for multiple multiplexers, which will allow you to fall back to a widely-supported multiplexer if a preferred
-choice is not supported by a remote peer.
+Stream multiplexing (or stream muxing) is a method to send multiple streams
+of data over a single communication link. It combines multiple signals into one
+unified signal. The unifed signal can then be demultiplexed (or demuxed), where
+the output becomes available to use in an application.
+
+Stream multiplexing is done to share a single TCP connection using unique port numbers
+that distinguish streams between multiple processes (such as kademlia and gossipsub)
+used by applications (such as IPFS) to make connection and transmission more efficient.
+
+## Muxers in libp2p
+
+Stream muxers are pluggable in libp2p. A given libp2p application can support several
+muxers simultaneously and choose one as the default, and allowing an application to
+fall back to a widely-supported multiplexer if a remote peer does not support a preferred
+choice.
+
+With muxing, libp2p applications can have separate communication streams between peers
+and multiple concurrent streams open simultaneously with a peer. Stream muxing
+allows peers to initialize and use the same [transport](../../transports/overview)
+connection across the lifetime of their interaction with another peer.
 
 {{< alert icon="💡" context="info">}}
-libp2p's multiplexing happens at the application layer, meaning it's not provided by the
-operating system's network stack. However, developers writing libp2p applications rarely need to
-interact with stream multiplexers directly, except during initial configuration to control which
-modules are enabled.
+libp2p's multiplexing happens at the application layer, meaning the
```

Comment:
That's only true for TCP, for QUIC and some other transports, stream multiplexing happens at the transport layer.